rpcserver.go

```diff
@@ -294,11 +294,21 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 func (r *rpcServer) MintAsset(ctx context.Context,
 	req *tarorpc.MintAssetRequest) (*tarorpc.MintAssetResponse, error) {
 
+	groupKey := &btcec.PublicKey{}
+	if req.EnableEmission && len(req.GroupKey) != 0 {
```

Comment:
Re the issue itself #110, is the plan to add support for grouping of _unminted_ assets in this PR? So eg: making 10 collectibles, want them to have the same group key, don't yet know the group key (and don't want to force the user to derive this value).

Reply:
This PR does not include the grouping for unminted assets, I figured this feature should land first & separately.